packages/config-cli/README.md

```diff
@@ -0,0 +1,18 @@
+# Basemaps Config CLI
+
+This package is to control the configuration in the LINZ basemaps product.
+
+## Usage -- Screenshots
+
+Dump the screenshots from basemaps production
+
+```bash
+./bin/bmc.js screenshot
```

Comment:
NIT: if its described as `bin : { "bmc": ... }` the actual command will be `bmc` once you npm install it